plugin-server/src/worker/ingestion/event-pipeline/normalizeEventStep.ts

```diff
@@ -0,0 +1,24 @@
+import { PluginEvent } from '@posthog/plugin-scaffold'
+import { DateTime } from 'luxon'
+
+import { normalizeEvent, normalizeProcessPerson } from '../../../utils/event'
+import { status } from '../../../utils/status'
+import { parseEventTimestamp } from '../timestamps'
+
+export function normalizeEventStep(event: PluginEvent, processPerson: boolean): [PluginEvent, DateTime] {
+    let timestamp: DateTime
+    try {
+        event = normalizeEvent(event)
+        event = normalizeProcessPerson(event, processPerson)
+        timestamp = parseEventTimestamp(event)
+    } catch (error) {
+        status.warn('⚠️', 'Failed normalizing event', {
+            team_id: event.team_id,
+            uuid: event.uuid,
+            error,
+        })
+        throw error
```

Comment:
What happens when this error is thrown?

Reply:
Hmm, the same thing as it always did before, this code just moved from the `processPersonsStep` because it wasn't related to People IMO.

I found no match for "Failed normalizing event" in the last 7 days.

If I'm reading the code right, because this doesn't throw something marked `isRetriable = false`, it will retry over and over. But more importantly, I think this try/catch is vestigial or defensive? I don't think this code actually throws. I could remove the try/catch, but if it ever did throw then it gives some extra info if this ever did throw.

We could make the case that we should DLQ here, but that is true of every other place in the pipeline... and I'm not sure we should make that a change as part of this PR?